distiller/config.py

```diff
@@ -137,13 +137,13 @@ def add_policy_to_scheduler(policy, policy_def, schedule):
                             frequency=policy_def['frequency'])
 
 
-def file_config(model, optimizer, filename):
+def file_config(model, optimizer, filename, start_epoch):
```

Comment:
This should be optional as well
```suggestion
def file_config(model, optimizer, filename, start_epoch=0):
```